docs/quickstart.rst

```diff
@@ -133,59 +217,61 @@ states it has assigned them:
 
 .. code-block:: text
 
+   # on the monitor virtual machine
+
+   sudo su - postgres
+   export PATH="/usr/lib/postgresql/11/bin/:$PATH"
+
    pg_autoctl show state --pgdata ./monitor
-        Name |   Port | Group |  Node |     Current State |    Assigned State
-   ----------+--------+-------+-------+-------------------+------------------
-   127.0.0.1 |   6010 |     0 |     1 |           primary |           primary
-   127.0.0.1 |   6011 |     0 |     2 |         secondary |         secondary
+       Name |   Port | Group |  Node |     Current State |    Assigned State
+   ---------+--------+-------+-------+-------------------+------------------
+   10.0.1.5 |   5432 |     0 |     1 |           primary |           primary
+   10.0.1.6 |   5432 |     0 |     2 |         secondary |         secondary
 
 This looks good. We can add data to the primary, and watch it get
 reflected in the secondary.
 
 .. code-block:: bash
 
+   # on your local machine
+
    # add data to primary
-   psql -p 6010 \
+   psql TODO -p 6010 \
      -c 'create table foo as select generate_series(1,1000000) bar;'
 
    # query secondary
-   psql -p 6011 -c 'select count(*) from foo;'
+   psql TODO -p 6011 -c 'select count(*) from foo;'
      count
    ---------
     1000000
 
 Cause a failover
 ----------------
 
-This plot is too boring, time to introduce a problem. We’ll run the
-primary out of disk space and watch the secondary get promoted.
+This plot is too boring, time to introduce a problem. We’ll turn off the
+primary and watch the secondary get promoted.
 
 In one terminal let’s keep an eye on events:
 
 .. code-block:: bash
 
    watch pg_autoctl show events --pgdata ./monitor
 
-In another terminal we’ll consume node A’s disk space and try to restart
-the database. It will refuse to start up.
+In another terminal we’ll turn off the virtual server.
 
 .. code-block:: bash
 
-   # postgres went to sleep one night and didn’t wake up…
-   pg_ctl -D /mnt/node_a/data stop &&
-     dd if=/dev/zero of=/mnt/node_a/bigfile bs=300MB count=1
-
-   # it will refuse to start back up with no free disk space
-   df /mnt/node_a
-   Filesystem     1K-blocks   Used Available Use% Mounted on
-   tmpfs             409600 409600         0 100% /mnt/node_a
+   az vm stop \
+     --resource-group ha-demo \
+     --name ha-demo-a
 
-After a number of failed attempts to restart node A, its keeper signals
-that the node is unhealthy and the node is put into the "demoted" state.
-The monitor promotes node B to be the new primary.
+After a number of failed attempts to talk to node A, the monitor determines
+the node is unhealthy and puts it into the "demoted" state.  The monitor
+promotes node B to be the new primary.
 
 .. code-block:: bash
 
+   TODO
```

Comment:
We should show the new state of the formation/group, and then reboot the first node and see it rejoin the cluster as a secondary I believe. Also, again, I think some copy/paste from the actual output of the pg_autoctl command would be nice to include here. There are many things that happen behind the scene and that we can introduce to the user, or at least have them realise they don't want to script all of that themselves and would rather just use our tool here.